lib/p2p/packets/packets.ts

```diff
@@ -0,0 +1,15 @@
+import PacketType from './PacketType';
+import OrderPacket from './OrderPacket';
+
+const fromRaw = (type:string, body:string) => {
```

Comment:
This file feels a bit out of place to me. I think the singular `packet` makes more sense, for consistency and since this only is representing/creating a single packet. But we already have a `Packet` class. I'm leaning towards getting rid of this file, and moving the `fromRaw` method to a static method on `Packet` (I checked and abstract classes can have static methods).

Reply:
I’m not in favor with putting it on `Packet` as a static method, because it will create circular dependencies. 

What we have here is basically a simple function for mapping value to instance. I thought of doing it by using a simple object, but left it as it is for now, in case more logic will be needed. 

So if the name is the problematic thing here, we can either:
* export it as a stand-alone function 
* export it under `packet`, and turn `Packet` to `BasePacket`

Reply:
Ah OK yeah I can envision the circular dependencies now that you mention it. I'd like your second bullet point, I think the naming will just be more clear. Also, on second look you should be able to declare a return type of `BasePacket` here.

Reply:
Yes, I added the return type now. 

After trying what I suggested, I didn't found calling it `packet` very good. It result with something like this:
`const packet = packet.fromRaw(...)` 
which I didn't even wanted to fix by different naming, because I found `packet` to be the better term for the instance.

So instead, I ended up calling it `packetUtils`